app/views/email_alert_subscriptions/new.html.erb

```diff
@@ -10,8 +10,21 @@
 <div class="outer-block">
   <div class="signup-content">
     <%= form_tag email_alert_subscriptions_path(subscriber_list_params: subscriber_list_params), class: 'signup-choices form' do %>
-
-      <% if @signup_presenter.can_modify_choices? %>
+      <% if @signup_presenter.hidden_choices.any? %>
+        <%= render partial: 'govuk_publishing_components/components/title', locals: {
+          title: @signup_presenter.name,
+          context: "Email alert subscription"
+        } %>
+        <div class="js-hidden">
+          <%= render "govuk_publishing_components/components/checkboxes", {
```

Comment:
Is there a reason why this isn't a hidden field? Using `js-hidden` to hide the elements will show 2 unintuitive checkboxes when js is disabled. Also there's no point in constructing the `hidden_choices` hash(es) if we aren't using them with these checkboxes.
I think we can make this a hidden field with
```ruby
<% hidden_choices.each do |hidden_choice| %>
  <%= hidden_field_tag hidden_choice[:name], hidden_choice[:value] %>
<% end %>
```